Dockerfile

```diff
@@ -1,31 +1,50 @@
 # Jupyter container used for Galaxy IPython (+other kernels) Integration
 
-# from 5th March 2021
+# from June 2023
 FROM jupyter/datascience-notebook:python-3.10
 
 MAINTAINER Björn A. Grüning, bjoern.gruening@gmail.com
 
-ENV DEBIAN_FRONTEND noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
 
-# Python packages
+# Set channels to (defaults) > bioconda > conda-forge
 RUN conda config --add channels conda-forge && \
-    conda config --add channels bioconda && \
+    conda config --add channels bioconda
+    #conda config --add channels defaults
+
+# Pre-installed mamba is raising a conda error:
+# "The environment is inconsistent"
+RUN conda remove mamba --yes
```

Comment:
Oh, this is unfortunate ...

Reply:
Is it worthwhile to try python-3.11 https://hub.docker.com/r/jupyter/datascience-notebook/tags?page=1&name=python-3.1

There is the potential that it creates other python issues or a few conda packages are not available.
What about updating mamba instead?

Reply:
I am essentially updating mamba, but conda update did not work smoothly. So I am deleting mamba here, and re-installing the newer version at line #39.

I already tried Python 3.11, it works out of the box. But some libs/programs might not yet be compatible with it, so I want to make a 3.10 release before 3.11.